cellxgene_schema_cli/cellxgene_schema/ontology_processing.py

```diff
@@ -0,0 +1,192 @@
+import owlready2
+import yaml
+import urllib.request
+import os
+import gzip
+import json
+
+
+def _dowload_owls(
```

Comment:
_dowload -> _download

Reply:
fixed in 4ac358decef7509c5944b70e93c9bd1e22eb45c1